scenedetect/cli/context.py

```diff
@@ -341,6 +341,13 @@ def process_input(self):
         # files with based on the given commands (list-scenes, split-video, save-images, etc...).
         cut_list = self.scene_manager.get_cut_list(base_timecode)
         scene_list = self.scene_manager.get_scene_list(base_timecode)
+
+        if self.drop_short_scenes:
+            scene_list = [
+                s for s in scene_list
+                if ( s[1] - s[0] ) >= self.min_scene_len
```

Comment:
Does this actually combine short scenes with their neighbors, or just drop them entirely?  I'm thinking maybe this PR should be split up into two, one to make `--min-scene-len` a top level option, and another which focuses on implementation of `--drop-short-scenes`... Thoughts?

Thanks for the PR, and sorry for the lengthy delay in getting to this.